test/integ/test_cursor.py

```diff
@@ -1318,6 +1318,64 @@ def test_resultbatch_lazy_fetching(conn_cnx, result_format, patch_path):
                 assert patched_download.call_count == 5
 
 
+@pytest.mark.skipolddriver(reason="new feature in v2.5.0")
+@pytest.mark.parametrize(
+    "result_format,patch_path",
+    (
+        ("json", "snowflake.connector.result_batch.JSONResultBatch.create_iter"),
+        ("arrow", "snowflake.connector.result_batch.ArrowResultBatch.create_iter"),
+    ),
+)
+def test_resultbatches_can_access_schema(conn_cnx, result_format, patch_path):
```

Comment:
This test seems to be mostly just a copy of `test_resultbatch_lazy_fetching`. Could we not just add the assert statement to the end of that test?